spec/pronto/runner_spec.rb

```diff
@@ -26,6 +26,21 @@ module Pronto
         it { should be_truthy }
       end
 
+      context 'ending with .gemspec' do
+        let(:path) { 'test.gemspec' }
+        it { should be_truthy }
+      end
+
+      context 'named Gemfile' do
+        let(:path) { 'test/Gemfile' }
```

Comment:
Since the follow spec tests the Gemfile in a directory, should this path just be `Gemfile`?